app/e2e/auth.setup.ts

```diff
@@ -8,6 +8,8 @@ test.beforeEach(async ({ page }) => {});
 
 setup("authenticate", async ({ page, request }) => {
   // log the user in
+  // First set timeout to accommodate send email to finish process
+  test.setTimeout(60000);
```

Comment:
Is this reliable? Can we wait async for the resource to be ready instead of always introducing a 60 second delay into our CI pipeline?